api/lib/source/polkadotV0-source.js

```diff
@@ -207,6 +207,7 @@ class polkadotAPI {
   async getBalancesV2FromAddress(address, fiatCurrency) {
     const api = await this.getAPI()
     const account = await api.query.system.account(address)
+    // See https://wiki.polkadot.network/docs/en/build-protocol-info#free-vs-reserved-vs-locked-vs-vesting-balance
```

Comment:
:sweat_smile:  I would include this but also say in plain English: "We need to account for funds that are locked on proposals for voting" or something along those lines

Reply:
For the lazy (guilty!) I included some considerations about balances :-)

Reply:
ok nice